Cargo.toml

```
@@ -14,7 +14,7 @@ crossbeam = "0.8"
 dashmap = "6"
 hashlink = "0.9"
 indexmap = "2"
-orx-concurrent-vec = "2"
+append-only-vec = "0.1"
```

```suggestion
append-only-vec = "0.1.5"
```

we actually need 0.1.5 since it contains [this change](https://github.com/droundy/append-only-vec/pull/4)